net.warsow.Warsow.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: net.warsow.Warsow
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: warsow
+rename-icon: warsow
+rename-desktop-file: warsow.desktop
+rename-appdata-file: warsow.appdata.xml
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - name: ImageMagick
+    config-opts:
+      - --disable-static
+      - --disable-docs
+      - --with-hdri
+      - --with-pic
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/7.1.0-33.tar.gz
+        sha256: dd18045614c19d11739311b207688c2cd480ed309d466685f1811f7a11df98ee
+        x-checker-data:
+          type: anitya
+          project-id: 1372
+          stable-only: true
+          url-template: https://github.com/ImageMagick/ImageMagick/archive/$version.tar.gz
+    cleanup:
+      - /bin
+      - /etc
+      - /lib/ImageMagick*
+      - /share/ImageMagick*
+
+  - name: warsow
+    buildsystem: cmake
+    no-make-install: true
+    subdir: source/source
+    config-opts:
+      - -DQFUSION_GAME=Warsow
+      - -DUSE_SDL2=ON
+    build-options:
+      cflags: -fcommon
+    post-install:
+      - install -Dm0755 build/warsow.* ${FLATPAK_DEST}/bin/warsow
+      - install -Dm0755 build/wsw_server.* ${FLATPAK_DEST}/bin/warsow-server
+      - install -Dm0755 build/wswtv_server.* ${FLATPAK_DEST}/bin/warsow-tv-server
+      - mkdir -p ${FLATPAK_DEST}/share/warsow/libs
+      - cp build/libs/*.so ${FLATPAK_DEST}/share/warsow/libs
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/
+      - convert -strip ../icons/warsow256x256.xpm ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/warsow.png
```

Comment:
I would prefer the .png in the repo and that it be submitted upstream rather that building ImageMagick just for that.

Reply:
Yes, it seems overkill.